jupyter-on-gke/iap_module/variables.tf

```diff
@@ -49,4 +49,16 @@ variable "client" {
     secret = string
   })
   description = "oauth client"
+}
+
+variable "url_domain_addr" { 
+  type = string
+  description = "Domain provided by the user. If it's empty, we will create one for you."
+  default = ""
+}
+
+variable "url_domain_name" { 
```

Comment:
What is name vs address? Where is name used?

Reply:
So the name is used by the ingress annotations. It's used to specify the global ip address.